examples/tutorials/matmul/cpu_matmul_quantization.cpp

```diff
@@ -203,7 +203,10 @@ int compare_vectors(const std::vector<float> &v1,
 
 } // namespace
 
-engine eng(engine::kind::cpu, 0); // We create a global engine for simplicity
```

Comment:
This engine is used only within a single translation unit isn't it?

Reply:
I believe so.

Reply:
Can you elaborate on the problem with it being global?

Reply:
Discussed offline, the order of destruction for internal singletons and related to them objects might expose issues.